Core/TimberApi/ToolGroupSystem/ToolGroupConfigurator.cs

```diff
@@ -0,0 +1,28 @@
+using Bindito.Core;
+using TimberApi.ConfiguratorSystem;
+using TimberApi.SceneSystem;
+using TimberApi.SpecificationSystem;
+using TimberApi.ToolGroupSystem.ToolGroups.ConstructionMode;
+using TimberApi.ToolGroupSystem.ToolGroups.Default;
+using TimberApi.ToolGroupSystem.ToolGroups.PlantingMode;
+using TimberApi.ToolGroupSystem.ToolGroups.TreeCuttingArea;
+
+namespace TimberApi.ToolGroupSystem
+{
+    [Configurator(SceneEntrypoint.InGame)]
```

Comment:
same as in bottombarconfigurator, maybe add mapeditor here so tools can be added to the mapeditro as well?

Reply:
Map editor does not work out of the box, needs more tools and maybe other stuff